src/thread/linux_raw.rs

```diff
@@ -76,6 +76,8 @@ impl Thread {
 /// This is not `repr(C)` and not ABI-exposed.
 struct ThreadData {
     thread_id: AtomicI32,
+    #[cfg(feature = "unstable-errno")]
+    errno_val: AtomicI32,
```

Comment:
Can `errno_val` be a regular `i32` instead of an `AtomicI32`? It's thread-local, and it isn't typically mutated atomically.

Reply:
I've turned it into a `Cell<i32>`.